src/controllers/controller.line.js

```diff
@@ -148,46 +165,33 @@ module.exports = DatasetController.extend({
 	/**
 	 * @private
 	 */
-	_resolveElementOptions: function(point, index) {
+	_resolveElementOptions: function(element, elementName, optionNameMap, index, scriptable) {
```

Comment:
I would not try to share this method between line and point elements (at least not yet) but instead duplicate it into `_resolveLineOptions` (and rename this one to `_resolvePointOptions`).